raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -256,7 +260,7 @@ private void maybeBecomeLeader(CandidateState state) throws IOException {
     }
 
     private void becomeCandidate() throws IOException {
-        electionTimer.reset(electionTimeoutMs);
+        electionTimer.reset(electionTimeoutMs + randomElectionJitterMs());
```

Comment:
We add the jitter to the election timeout upon becoming a candidate. This was really just to avoid the need for another timer.